plugins/permission-node/report.api.md

```diff
@@ -268,6 +350,13 @@ export type PermissionRule<
   toQuery(params: NoInfer_2<TParams>): PermissionCriteria<TQuery>;
 };
 
+// @public
+export type PermissionRuleAccessor<
```

Comment:
Naming suggestion: `PermissionRuleset` with `getByName`

Reply:
Yep good call on wrapping this up a bit, will refactor. Naming-wise I like ruleset more, I'll switch to that for now.

Reply:
Updated. I went with `getRuleByName` to be a bit more explicit and to have the type itself be a bit more strict since I don't think it's worth making it an opaque type at this point.